src/funding/venmo/config.jsx

```diff
@@ -21,6 +22,11 @@ export function getVenmoConfig() : FundingSourceConfig {
         ],
         
         eligible: ({ experiment }) => {
+            // If user is using a tablet don't show venmo button
+            if (isTablet()) {
```

Comment:
I think this check can be removed. The `isTablet()` check in util.js will make `experiment.enableVenmo` false already.
```suggestion
```